magento/magento2
# Redirect to "Options" config path more accurate (in Currency Rate form)

### Description (*)

Redirect to "Currency Option Path" more accurate (Auto open the collapse)

### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. Go to backend, Stores->Currency Rates
2. Click "Options"

![image](https://user-images.githubusercontent.com/8234209/88479754-a5ee3400-cf7b-11ea-8883-4e5cb878eb15.png)

=> Should be redirect to currency options in System config and automatically open the collapse like that:

![image](https://user-images.githubusercontent.com/8234209/88479773-c4542f80-cf7b-11ea-8592-2b9e33874a4b.png)

(Now, it is not opened)

![image](https://user-images.githubusercontent.com/8234209/88479782-d8982c80-cf7b-11ea-98bf-f91e4016a0ce.png)


### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29336: Redirect to "Options" config path more accurate (in Currency Rate form)
